### ISSUE TITLE
useEffect is only firing on initial render but not on later updates

### ISSUE THREAD

**Issue by zanaptak** [Zaid-Ajaj/Feliz#72]
`useEffect` is not firing on every update. I believe it is supposed to by default -- [React docs here](https://reactjs.org/docs/hooks-effect.html).

Here is repro code -- the Fable.React version fires on every increment rerender, but the Feliz version only fires on the initial render:

```fs
let counter =
    React.functionComponent("Counter", fun () ->
        let (count, setCount) = React.useState(0)

        // prints on every increment
        Fable.React.HookBindings.Hooks.useEffect ( fun _ -> printfn "useEffect from Fable.React" )

        // prints only on initial render
        Feliz.React.useEffect ( fun _ -> printfn "useEffect from Feliz.React" )

        Html.div [
            Html.h1 count
            Html.button [
                prop.text "Increment"
                prop.onClick (fun _ -> setCount(count + 1))
            ]
        ]
    )
```

**Comment by cmeeren** [comment]
Sounds like it might be a duplicate of #70, have you tested with the most recent version?

**Comment by zanaptak** [comment]
Issue still exists in 0.65.0.

**Comment by Zaid-Ajaj** [comment]
Hmm looks like the default React.useEffect delegates an empty dependency list to the internal API, can you try the overload that relies on a dependency: 

Feliz.React.useEffect(<callback>, count) 

We need to change the default useEffect to reflect React expected behaviour, A PR would be highly appreciated

**Comment by Zaid-Ajaj** [comment]
I have published version v0.65.1 with your fix from #73 can you take it for a spin? Please re-open the issue if the problem persists
